queries.sql

```diff
@@ -50,4 +50,19 @@ SELECT animals.name, owners.full_name FROM animals RIGHT JOIN owners ON animals.
 SELECT species.name, COUNT(animals.species_id) FROM animals JOIN species ON species.id = animals.species_id GROUP BY species.name;
 SELECT animals.name from animals JOIN owners ON owners.id = animals.owner_id JOIN species on species.id = animals.species_id WHERE animals.species_id = '2' AND animals.owner_id = '2';
 SELECT animals.name from animals JOIN owners ON owners.id = animals.owner_id WHERE animals.escape_attempts = '0' AND animals.owner_id = '5';
+SELECT full_name, COUNT(owner_id) FROM owners JOIN animals on owners.id = animals.owner_id GROUP BY full_name ORDER BY COUNT (owner_id) desc limit 1;
+
+
+/* Day 4 */
+
+SELECT animals.name FROM animals JOIN visits ON animals.id = visits.animal_id WHERE vet_id = '1' ORDER BY visits.date_of_visit desc limit 1;
+SELECT COUNT(DISTINCT animal_id) FROM visits WHERE vet_id = '2';
+SELECT vets.name, specialization.vet_id, specialization.species_id FROM vets LEFT JOIN specialization ON specialization.vet_id = vets.id;
+SELECT animals.name , visits.date_of_visit FROM animals JOIN visits ON animals.id = visits.animal_id WHERE visits.date_of_visit BETWEEN '2020-04-01' AND '2020-08-30' AND vet_id = '3';
```

Comment:
- [ ] according to the requirements you should List all vets and their specialties, including vets with no specialties as well.